### PR TITLE
(maint) Fix grammatical issue in pending reboot warning message

### PR DESCRIPTION
**Pull request by sorousherafat**
## Description Of Changes
Changed 'Pending Reboot' warning message to be more accurate grammatically.

## Change Types Made

* [ ] Bug fix (non-breaking change).
* [x] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.

## Change Checklist

* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [x] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

## Related Issue

Fixes no issue

### CODE REVIEW

File: src/chocolatey/infrastructure.app/validations/SystemStateValidation.cs [hunk 1]
```diff
@@ -69,7 +69,7 @@ private void check_system_pending_reboot(ChocolateyConfiguration config, ICollec
                     validationResults.Add(new ValidationResult
                     {
                         Message = @"A pending system reboot request has been detected, however, this is
-   being ignored due to the current command being used '{0}'.
+   being ignored due to the current command used being '{0}'.
```
Comment (pauby) [inline src/chocolatey/infrastructure.app/validations/SystemStateValidation.cs:72]:
```suggestion
   being ignored due to the current command '{0}' being used.

```

**Review by pauby — CHANGES_REQUESTED** [review]
I feel this reads better.

File: src/chocolatey/infrastructure.app/validations/SystemStateValidation.cs (hunk 1 above)
Comment (gep13) [reply src/chocolatey/infrastructure.app/validations/SystemStateValidation.cs]: Yes, I agree, I will get this fixed up.

**Comment by gep13** [comment]
@sorousherafat thank you very much for taking the time to raise this PR to highlight this problem.  After discussing with @pauby we are going to move forward with his suggestion, and I have taken the liberty to update this PR with his suggestions.

**Comment by gep13** [comment]
Not sure why the CLA check is being marked as not there:

image

but it has been signed, as per this comment:

https://github.com/chocolatey/choco/pull/3164#issuecomment-1555878866

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**